client/src/partials/reports/global_transaction/global_transaction.js

```diff
@@ -77,22 +80,19 @@ angular.module('bhima.controllers')
         datef : util.sqlDate($scope.dates.from),
         datet : util.sqlDate($scope.dates.to)
       };
+      var url = '/reports/allTransactions/?source=' + $scope.model.source_id +'&enterprise_id=' + $scope.enterprise.id + '&account_id=' + $scope.model.account_id + '&datef=' + util.sqlDate($scope.dates.from) + '&datet=' + util.sqlDate($scope.dates.to);
```

Comment:
This is much better!  It will be more readable if you use multiple lines:
```javascript
var url = '/reports/allTransactions/?source=' + $scope.model.source_id +
   '&enterprise_id=' + $scope.enterprise.id + 
   '&account_id=' + $scope.model.account_id + 
   '&datef=' + util.sqlDate($scope.dates.from) + 
   '&datet=' + util.sqlDate($scope.dates.to);
```